### PR TITLE
New plugins

### PR DESCRIPTION
**Pull request by Tomsod**
Hello!
I made a few new plugins for your thingy, if you're interested.
Those are: CPU load, disk usage and wireless connection stats.
I did make them mainly for myself, but I reckon the code's tidy enough
for me to share it. There's also one bugfix somewhere.

### CODE REVIEW

File: fsinfo/src/fsinfo.c [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+            section->path = NULL;
+          }
+      }
+    if (!section->path)
+      {
+        FILE *mtab = setmntent(context->mtab, "r");
+        if (!mtab)
+            UPDATE_ERROR("Could not open mtab");
+        struct mntent *mnt;
+        while (mnt = getmntent(mtab))
```
Comment (sardemff7) [inline fsinfo/src/fsinfo.c]: GCC warning: you should check for `!= NULL` to make clear what your code does.

**Comment by sardemff7** [comment]
I merged this, please do a follow-up commit to fix the warning.
And remove myself as the man pages author too.

I’ll give you commit access to maintain your plugins.
Thanks a lot for them!

**Comment by Tomsod** [comment]
All right, I'll try to be vigilant for bug reports.
(Don't forget to give me push access, though!)

About AUTHORS man section, though, I think it's supposed to list
authors of the program, not of the man page itself. So it's okay.

**Comment by sardemff7** [comment]
Rights should be ok now.

About authors, **you** are the author of these three plugins, so I think it is better to have you only listed in these man pages. Also, it gives people the entry point for questions and bug reports, which I am not since I do not know how they work exactly. At least put me last in the list so that we can automate some stuff based on this list. :-)

**Pull request closed without merging**